samples/bot-types-of-cards/nodejs/.env

```diff
@@ -0,0 +1,2 @@
+MicrosoftAppId="{{Microsoft-App-Id}}"
+MicrosoftAppPassword="{{Microsoft-App-Password}}"
```

Comment:
Make sure that these placeholders are properly mentioned in your readme file.

Reply:
fixed